spec/unit/entangler_spec.rb

```diff
@@ -59,6 +59,23 @@
       @entangler.entangle.must_include strip(ddl)
     end
 
+    it 'should retry trigger creation when it hits a lock wait timeout' do
```

Comment:
What would you think about having three tests?

1. Test that the LHM is successful with _NO_ locks at all
2. Test that the LHM is successful with more than one, but  less than MAX_RETRIES lock timeouts
3. Test that the LHM fails with more than MAX_RETRIES locks.